WcaOnRails/app/webpacker/components/Panel/Leader/GroupsManager.jsx

```diff
@@ -108,30 +146,30 @@ export default function GroupsManager({ loggedInUserId }) {
     'groupName', // Sort params
     { isActive: true, isGroupHidden: false, status: 'leader' },
   ));
-  const [selectedGroupId, setSelectedGroupId] = useState();
+  const [selectedGroupIndex, setSelectedGroupIndex] = useState();
 
   useEffect(() => {
-    setSelectedGroupId(roles?.[0]?.group.id);
+    if (roles?.length > 0) setSelectedGroupIndex(0);
   }, [roles]);
 
   if (!loading && roles.length === 0) return <p>You are not a leader of any groups.</p>;
-  if (loading || selectedGroupId === undefined) return <Loading />;
+  if (loading || selectedGroupIndex === undefined) return <Loading />;
   if (error) return <Errored />;
 
   return (
     <>
       <div>
         <Dropdown
-          options={roles.map((role) => ({
+          options={roles.map((role, index) => ({
             key: role.id,
             text: role.group.name,
-            value: role.group.id,
+            value: index,
           }))}
-          value={selectedGroupId}
-          onChange={(_, { value }) => setSelectedGroupId(value)}
+          value={selectedGroupIndex}
+          onChange={(_, { value }) => setSelectedGroupIndex(value)}
```

Comment:
Can be simplified via `useInputState`, see my other comment in this PR.

Reply:
Done